packages/components/input/src/textarea.tsx

```diff
@@ -139,8 +151,21 @@ const Textarea = forwardRef<"textarea", TextAreaProps>(
     return (
       <Component {...getBaseProps()}>
         {shouldLabelBeOutside ? labelContent : null}
-        <div {...getInputWrapperProps()} data-has-multiple-rows={dataAttr(hasMultipleRows)}>
-          {shouldLabelBeInside ? labelContent : null}
+        <div
+          {...getInputWrapperProps({
+            style: {flexDirection: "column", alignItems: "flex-start", gap: "0"},
```

Comment:
move the style classes to `getInputWrapperProps`

Reply:
if move the style to getInputWrapperProps ,the input component is also affected

Reply:
there is a variable called `isMultiline`. if it's `true`, then it's textarea, else input.